presto-main/src/test/java/com/facebook/presto/sql/planner/iterative/rule/TestSimplifyExpressions.java

```diff
@@ -188,7 +139,7 @@ public Expression rewriteLogicalBinaryExpression(LogicalBinaryExpression node, V
         {
             List<Expression> predicates = extractPredicates(node.getType(), node).stream()
                     .map(p -> treeRewriter.rewrite(p, context))
-                    .sorted((p1, p2) -> p1.toString().compareTo(p2.toString()))
+                    .sorted(Comparator.comparing(Expression::toString))
```

Comment:
unrelated?

Reply:
It's just cleaning (with no semantic change) in the file I already change a lot. Not worth for another PR, IMO.